### ISSUE TITLE
DeiT depth 24 (CaiT - TABLE 1)

### ISSUE THREAD

**Issue by GoJunHyeong** [facebookresearch/deit#218]
Hello. I would like to re-implement table 1 of the Cait paper. When learning by modifying the stochastic depth ratio d=0.05 (depth12) to 0.2 (depth24) in the learning setting of DeiT-S-12 depth, a nan error occurs. I used the code of the repository as it is. What methods should be used to re-implement DeiT-S 24 depth?

**Comment by TouvronHugo** [comment]
Hi @GoJunHyeong ,
Thanks for your message.
Could you provide the hparams you are using for this run?
Best,
Hugo

**Comment by GoJunHyeong** [comment]
Hello, @TouvronHugo, Thank you for your reply.

I used the official training script of DeiT-S.
[https://github.com/facebookresearch/deit/blob/main/README_deit.md](url)

To summarize I use:

Batch size: 1024 (8 GPUs)
stochastic depth ratio : 0.2
dropout ratio : 0
lr: 5e-4
epochs: 300
weight-decay: 0.05
sched: cosine
input size: 224
smoothing: 0.1
repeated_aug: True
mixup: 0.8
cutmix: 1.0
warmup epochs: 
rand aug: rand-m9-mstd0.5-inc1
erase prob: 0.25
Optimizer: AdamW

There is no layer scale or scalar alpha weighting scheme.

The accuracy mentioned in the paper is 81.0, but I achieved 79.7 in my case.

Thank you.
